### PR TITLE
Fixes from comments after last edition of the course

### LINKED ISSUE #17: Comments - 3rd HEP C++ Course
I open this issue to collect all comments which cannot be done via PRs and were collected during the Aug HEP C++ Course.

- [x] function argument defaults are not mentioned yet. It may make sense to mention them during the first day in a slide.

### PR DESCRIPTION
**Pull request by sponce**
fixes #17

### CODE REVIEW

**Review by hageboeck — APPROVED** [review]
LGTM (but I didn't check every string for correctness whether it would compile).

**Comment by sponce** [comment]
@hageboeck this is now final if you want to have a last quick check

**Comment by sponce** [comment]
Did a wrong push and reverted to previous state. Sorry for the noise

File: talk/morelanguage.tex [hunk 1]
```diff
@@ -529,8 +549,8 @@
     \begin{cppcode*}{}
       T a;
       T b = a;      // 1. Copy assign
-      T c = T(2);   // 2. Move assign
-      T d = func(); // 3. Move assign
+      T Func() { ... }
```
Comment (hageboeck) [inline talk/morelanguage.tex:552]:
```suggestion
      T func() { ... }
```

**Review by hageboeck — APPROVED** [review]
LGTM! Just this one typo.

File: talk/morelanguage.tex (hunk 1 above)
Comment (sponce) [reply talk/morelanguage.tex]: Thanks for spotting it

**Pull request merged**